pkg/rewrite/rewrite.go

```diff
@@ -101,7 +101,8 @@ func Rewrite(rewriteOptions RewriteOptions) error {
 		IsOpenShift:          k8sutil.IsOpenShift(clientset),
 		IsGKEAutopilot:       k8sutil.IsGKEAutopilot(clientset),
 	}
-	if err := upstream.WriteUpstream(u, writeUpstreamOptions); err != nil {
+	err = upstream.WriteUpstream(u, writeUpstreamOptions)
```

Comment:
nit pick: i'd the previous format:
```suggestion
	if err := upstream.WriteUpstream(u, writeUpstreamOptions); err != nil {
```